docker-compose/.env

```diff
@@ -1,29 +1,19 @@
-TITLE="MariaDB CI"
-TITLE_URL=https://github.com/MariaDB/server
-BUILDMASTER_URL=https://buildbot.mariadb.org/
-BUILDMASTER_WG_IP=100.64.100.1
-MQ_ROUTER_URL=ws://127.0.0.1:8080/ws
-MASTER_PACKAGES_DIR="/mnt/autofs/master_packages"
-MASTER_CREDENTIALS_DIR="/srv/buildbot/master/master-credential-provider"
-GALERA_PACKAGES_DIR="/mnt/autofs/galera_packages"
-ARTIFACTS_URL="https://ci.mariadb.org"
-NGINX_ARTIFACTS_VHOST="ci.mariadb.org"
-NGINX_BUILDBOT_VHOST="buildbot.mariadb.org"
-NGINX_CR_HOST_WG_ADDR="100.64.100.20:8080"
-ENVIRON="PROD"
-BRANCH="main"
-MASTER_NONLATENT_DOCKERLIBRARY_WORKER="bb-rhel8-docker"
-MASTER_NONLATENT_BINTARS_WORKERS = '
-    {
-    "ro-apexis-bbw03-x64": {
-        "max_builds": 2,
-        "jobs": 12
-        },
-    "bg-bbw1-x64": {
-        "max_builds": 1,
-        "jobs": 12
-        }
-    }'
-MASTER_NONLATENT_BINTARS_VM_PORT="10000"
-MASTER_NONLATENT_BINTARS_WORKER_PORT="10007"
-CONTAINER_REGISTRY_URL="quay.io/mariadb-foundation/bb-worker:"
+TITLE='MariaDB CI'
+TITLE_URL='https://github.com/MariaDB/server'
+BUILDMASTER_URL='https://buildbot.mariadb.org/'
+BUILDMASTER_WG_IP='100.64.100.1'
+MQ_ROUTER_URL='ws://127.0.0.1:8080/ws'
+MASTER_PACKAGES_DIR='/mnt/autofs/master_packages'
+MASTER_CREDENTIALS_DIR='/srv/buildbot/master/master-credential-provider'
+GALERA_PACKAGES_DIR='/mnt/autofs/galera_packages'
+ARTIFACTS_URL='https://ci.mariadb.org'
+NGINX_ARTIFACTS_VHOST='ci.mariadb.org'
+NGINX_BUILDBOT_VHOST='buildbot.mariadb.org'
+NGINX_CR_HOST_WG_ADDR='100.64.100.20:8080'
+ENVIRON='PROD'
+BRANCH='main'
+MASTER_NONLATENT_DOCKERLIBRARY_WORKER='bb-rhel8-docker'
+MASTER_NONLATENT_BINTARS_WORKERS='{ "ro-apexis-bbw03-x64": { "max_builds": 2, "jobs": 12 }, "bg-bbw1-x64": { "max_builds": 1, "jobs": 12 } }'
```

Comment:
Yep, this is needed to make it a valid BASH file. Later we should get rid of this. I had to do a similar change in my refactoring work.